README.md

```diff
@@ -1,3 +1,21 @@
+DISCLAIMER
```

Comment:
You might want to have this disclaimer in your own fork of the repo but it does not belong in this upstream one.

Reply:
Hello @quinot 

As it was never reviewed/merged,  i even forgot of that opened PR, you can ignore it.